### ISSUE TITLE
Palguard issue

### ISSUE THREAD

**Issue by faisalmallick** [SwirlClinic/palworld_docker#1]
### **I am using your docker configuration and the server is running & I can play but I can not use Palguard mod. Is there any way to use Palguard mod in your docker configuration?**
Note: Palguard mod link: https://www.nexusmods.com/palworld/mods/451?tab=description

**Comment by SwirlClinic** [comment]
Yeah it's not working for me either. The author mentioned it doesn't work for Proton as well.
<img width="1014" alt="image" src="https://github.com/SwirlClinic/palworld_docker/assets/10433370/d3978166-226e-4c37-8d0a-5cd1705ec491">
